File: Niveau 2/4 – Fonctions/7 - Distance euclidienne.py
from math import *
def distance(x1, x2, y1, y2):
   return round(sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2)), 6)
x1 = float(input())
y1 = float(input())
x2 = float(input())
y2 = float(input())
print(distance(x1, x2, y1, y2))

File: Niveau 2/4 – Fonctions/7 - Distance euclidienne.cpp
#include <iostream>
#include <cmath>
using namespace std;

double distance(double x1, double y1, double x2, double y2)
{
  double dx = pow(x2 - x1, 2);
  double dy = pow(y2 - y1, 2);
  return sqrt(dx + dy);
}

int main()
{
  double x1, y1, x2, y2;
  cin >> x1 >> y1 >> x2 >> y2;
  cout << distance(x1, y1, x2, y2);
}